app/src/main/java/org/mozilla/focus/ext/String.kt

```diff
@@ -81,3 +83,13 @@ fun String?.toUri(): Uri? = if (this == null) {
 } else {
     Uri.parse(this)
 }
+
+fun String?.toJavaURI(): URI? = if (this == null) {
+    null
+} else {
+    try {
+        URI(this)
```

Comment:
Should we use URI for the other stuff? I picked it arbitrarily, so there's no harm in changing from Uri to URI. 

lolwow

Reply:
I don't actually know the differences – I think we should decide on a standard though, but I just wanted to use this code as is because I chose `URI` for a reason when I originally wrote it (maybe that's the way it was before though x_x).